portfolio-webapp/src/App.css

```
@@ -599,3 +639,27 @@ body {
   word-wrap: break-word;
   border: 1px solid #ccc;
 }
+
+.dm-sans {
+  font-family: "DM Sans", sans-serif;
+  font-optical-sizing: auto;
+  font-weight: 100;
+  font-style: normal;
+}
+
+
+
+input[type="checkbox"],
+input[type="radio"] {
+  accent-color: #28a745;
+}
+
+.checkbox-group {
+  align-self: center;
+  padding-top: 30px;
+}
+
+.bullet:before {
+  content: url('/bullet.svg');
```

Same thing with caching:
```suggestion
  content: url("../public/bullet.svg");
```